src/gameplay/game.ts

```diff
@@ -1255,7 +1257,11 @@ class Game extends Room {
     }
 
     // From this point on, no more game moves can be made. Game is finished.
-    gamesPlayedMetric.inc(1, { status: 'finished' });
+    console.log(this.roomCreationType);
+    gamesPlayedMetric.inc(1, {
+      status: 'finished',
+      game_type: roomCreationTypeStrToMetricLabel(this.roomCreationType),
```

Comment:
Instead of this `roomCreationTypeStrToMetricLabel`, let's make a function called 

`const function GetGameMetricType(game: Game)` 

that takes in a game object and returns the enum for the metric.